app/backend/lookuptool.py

```diff
@@ -1,13 +1,13 @@
-from os import path
 import csv
+from pathlib import Path
 from langchain.agents import Tool
 from langchain.callbacks.manager import Callbacks
 from typing import Optional
 
 class CsvLookupTool(Tool):
     data: dict[str, str] = {}
 
-    def __init__(self, filename: path, key_field: str, name: str = "lookup", 
+    def __init__(self, filename: str | Path, key_field: str, name: str = "lookup", 
```

Comment:
`path` is not a type. `open()` accepts both str and Path from path lib, so this is a more obvious annotation of what the method is expecting